modules/creating-a-data-science-project.adoc

```diff
@@ -4,16 +4,15 @@
 = Creating a data science project
 
 [role='_abstract']
-To start your data science work, create a data science project. Creating a project helps you organize your work in one place. You can also enhance your data science project by adding the following functionality:
+To implement a data science workflow, you must create a project. In OpenShift, a project is a Kubernetes namespace with additional annotations, and is the central method by which access to resources for regular users is managed. A project organizes your data science work in one place, keeping your content in isolation while also allowing collaboration with other developers and data scientists in your organization.
```

Comment:
Could simplify/make this section more active:

- 'and is the central method by which access to resources for regular users is managed' > 'the main way that you can manage user access to resources.'
- 'A project organizes your data science work in one place, keeping your content in isolation while also allowing collaboration with other developers and data scientists in your organization' > 'A project organizes your data science work in one place where you can collaborate with other developers and data scientists in your organization'